.github/workflows/validate.yml

```diff
@@ -0,0 +1,66 @@
+name: validate
+on: [push, pull_request]
+jobs:
+  main:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest]
```

Comment:
Let's go with ubuntu-latest only

Reply:
This is a package that's doing cross-platform things, so maybe we should keep it? 🤔

Reply:
Oh, that's a good point 👍